pkg/isbsvc/jetstream_service.go

```diff
@@ -21,6 +25,21 @@ type jetStreamSvc struct {
 	js       *jsclient.JetStreamContext
 }
 
+func (jss *jetStreamSvc) CreateWatermarkFetcher(ctx context.Context, bufferName string) (fetch.Fetcher, error) {
```

Comment:
Nit: move this function after `NewISBJetStreamSvc(`, let it sit together with other interface functions.

Reply:
Sure, addressed.